modules/dnn/misc/java/src/cpp/dnn_converters.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "dnn_converters.hpp"
 
+#define LOG_TAG "DNN"
```

Comment:
Used?
"org.opencv.dnn"

Reply:
Used in `Mat_to_MatShape` which is not used anywhere

Reply:
OK, please rename "org.opencv.dnn" (to keep tags consistent with other code)